### PR TITLE
discordtrlist

### PR DESCRIPTION
**Pull request by samet99**
Türkiyenin en iyi bot listesi

### CODE REVIEW

File: discordtrlist.xyz/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    },
+    "service": "TR List",
+    "description": {
+        "tr": "Türkiyenin en gelişmiş listi.",
+         "en": "Turkey's most advanced bot list"
+    },
+    "url": "discordtrlist.xyz",
+    "version": "1.1",
+    "logo": "https://cdn.discordapp.com/avatars/540077622202335233/f9cd6e9433d58ca9a84566cf3c242ef9.png",
+    "thumbnail": "https://cdn.discordapp.com/avatars/54007762220233523/f9cd6e9433d58ca9a84566cf3c242ef9.png",
```
Comment (Bas950) [inline discordtrlist.xyz/dist/metadata.json]: Upload you images somewhere else like imgur, do NOT use discord's cdn.

**Comment by Bas950** [comment]
Also upload proof that it works.

File: discordtrlist.xyz/dist/metadata.json (hunk 1 above)
Comment (samet99) [reply discordtrlist.xyz/dist/metadata.json]: Fix

**Comment by samet99** [comment]
> Also upload proof that it works.

I added

**Comment by Bas950** [comment]
image

**Comment by Bas950** [comment]
You presence must also include a large image key.

File: discordtrlist.xyz/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+    "author": {
+        "name": "A1D1+",
+        "id": "538049764592320543"
+    },
+    "service": "TR List",
```
Comment (Bas950) [inline discordtrlist.xyz/dist/metadata.json]: service name isnt the same as folder name.

**Comment by samet99** [comment]
Fix

**Pull request merged**